### PR TITLE
Also push container images to github / gcr.io

### PR DESCRIPTION
**Pull request by Firefishy**
This adds the support to also push container images to github's gcr.io

The gorelease code change has not yet been tested.

### CODE REVIEW

**Comment by tlimoncelli** [comment]
Thanks for writing this PR.  I'll merge it before the next release.

**Comment by jauderho** [comment]
This does not seem correct. gcr.io *and* ghcr.io both exist unfortunately but represent completely different registries.

* ghcr.io is GitHub Container Registry
* gcr.io is Google Container Registry

File: .goreleaser.yml [hunk 1]
```diff
@@ -120,12 +120,23 @@ docker_manifests:
       - *amd_image
       - *386_image
       - *arm_image
+  - name_template: "gcr.io/stackexchange/{{.ProjectName}}:{{ .Version }}"
```
Comment (jauderho) [inline .goreleaser.yml:123]: This should be ghcr.io instead.

File: .goreleaser.yml [hunk 2]
```diff
@@ -120,12 +120,23 @@ docker_manifests:
...
+      - *amd_image
+      - *386_image
+      - *arm_image
   - name_template: "stackexchange/{{.ProjectName}}:latest"
     skip_push: auto
     image_templates:
       - *amd_image
       - *386_image
       - *arm_image
+  - name_template: "gcr.io/stackexchange/{{.ProjectName}}:latest"
```
Comment (jauderho) [inline .goreleaser.yml:134]: This should be ghcr.io instead.

**Comment by tlimoncelli** [comment]
@Firefishy Could you re-open this PR and make @jauderho 's corrections?

**Comment by Firefishy** [comment]
Yes, will do! Sorry about the mistake.

**Pull request merged**